zeth_contracts/contracts/Groth16AltBN128MixerBase_test.sol

```diff
@@ -32,40 +31,40 @@ contract Groth16AltBN128MixerBase_test is Groth16AltBN128Mixer
         public
         returns (uint256)
     {
-        return hash_public_proof_data(public_data);
+        return hashPublicProofData(public_data);
```

Comment:
Should the method names in this file (e.g. `hash_public_proof_data_test`) also be updated?

Reply:
Oh yes, definitely forgot these ones. Thanks @dtebbs